packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.base.tsx

```diff
@@ -46,7 +54,6 @@ const DEFAULT_STRINGS: IDatePickerStrings = {
   prevYearAriaLabel: 'Go to previous year',
   nextYearAriaLabel: 'Go to next year'
 };
-
```

Comment:
Nitpicky as all hell I realize, but can you put this blank line before the back? Separating declarations like the class and this interface really help with scan-ability